front/components/vaults/VaultResourcesList.tsx

```diff
@@ -275,14 +341,33 @@ export const VaultResourcesList = ({
             isAdmin={isAdmin}
           />
         )}
-        {(category === "folder" || category === "website") && (
-          <EditVaultStaticDataSourcesViews
-            owner={owner}
-            plan={plan}
-            vault={vault}
-            category={category}
-            dataSources={dataSources}
-          />
+        {isWebsiteOrFolder(category) && (
+          <>
+            <EditVaultStaticDataSourcesViews
+              owner={owner}
+              plan={plan}
+              vault={vault}
+              category={category}
+              dataSources={dataSources}
+            />
+            {selectedDataSourceView && (
+              <>
+                <VaultFolderOrWebsiteModal
+                  isOpen={showFolderOrWebsiteModal}
+                  setOpen={(isOpen) => setShowFolderOrWebsiteModal(isOpen)}
```

Comment:
I don't know if we have a convention but in most dialog we are using onClose() (with sometimes a boolean to tell if some changes were saved)